src/sentry/static/sentry/app/constants/index.jsx

```diff
@@ -140,3 +140,9 @@ export const DEFAULT_RELATIVE_PERIODS = {
 // Special Search characters
 export const NEGATION_OPERATOR = '!';
 export const SEARCH_WILDCARD = '*';
+
+// Algolia documentation searchl
+export const ALGOLIA_APP_ID = 'OOK48W9UCL';
+export const ALGOLIA_READ_ONLY = '2d64ec1106519cbc672d863b0d200782';
```

Comment:
Should this be used on on-premise installs as well?

Reply:
Yeah, I don't see any reason it shouldn't.